articles/Non-custodial-Lightning-Network-Solutions.md

```diff
@@ -0,0 +1,36 @@
+| Author | Phoenix Green |
+| :---- | :---- |
+| Title | Non-custodial Lightning Network Solutions (Better Title needed) |
+| Headline  | featured text |
+| Cypherpunktimes | cypherpunktimes.com/...link |
+| Publish Date | 00 Jan 20?? |
+
+| Social Media | Link |
+| :---- | :---- |
+| Twitter | post link… |
+| Twitter | thread link… |
+| Twitter | poll link… |
+| Reddit  | link… |
+| Bison Relay | News feed title |
+| Other platforms | Instagram and TikTok |
+
+Thesis statement 
+
+The Lightning Network has achieved the monumental target of processing thousands of micro-transactions per second at the cost of a single Atom/Satoshi. But is it possible to have this functional in a non-custodial wallet?
+
+Pre-discussion, research and questions:
+* What are the most popular LN wallets?
```

Comment:
I like the topic. Maybe you should introduce a little about the lightning network revolution and why this technology is important. 
Also maybe is a good idea to list some projects that already implemented it